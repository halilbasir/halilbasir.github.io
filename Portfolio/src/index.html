<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Portfolio</title>

	<link rel="stylesheet" href="css/bootstrap.css">
	<link rel="stylesheet" href="css/style.css">
	<link
	  rel="stylesheet"
	  href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"
	/>
	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Open+Sans:wght@400;600&amily=Raleway:wght@700;800;900&display=swap" rel="stylesheet">
	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
	
</head>
<body>

	<header class="header" id="header">
		<div class="header-icons">
			<a class="header-icon"
				href="https://vk.me/h.basirov">
				<i class="fab fa-vk"></i>
			</a>

			<a class="header-icon"
				href="https://api.whatsapp.com/send?phone=79872533419" target="_blank">	
					<i class="fab fa-whatsapp"></i>
			</a>

			<a class="header-icon "
				href="https://t.me/HalilBasirov" target="_blank">
				<i class="fab fa-telegram-plane"></i>
			</a>	
		</div>
		<nav class="nav" id="nav">
			<a href="#" class="nav-link" data-scroll="#about" id="nav_link" >Обо мне</a>
			<a href="#" class="nav-link" data-scroll="#skills" id="nav_link">Навыки</a>
			<a href="#" class="nav-link" data-scroll="#process" id="nav_link">Процесс верстки</a>
			<a href="#" class="nav-link" data-scroll="#works" id="nav_link">Портфолио</a>
			<a href="#" class="nav-link" data-scroll="#contacts" id="nav_link">Контакты</a>
		</nav>

		<button class="nav-toggle" id="nav_toggle" type="button">
			<span class="nav-toggle__item">Menu</span>
		</button>

	</header>
	
	<main class="main" id="main">
		<div class="container">
			<div class="main-inner">
				<div class="row">
					<div class="col-lg-12 col-lg-auto">
						<h3 class="main-subtitle wow animate__animated animate__bounceInLeft">HTML / CSS / JS</h3>
						<h1 class="main-title wow animate__animated animate__bounceInRight">ДОБРЫЙ ДЕНЬ <br> Я ХАЛИЛЬ БАСИРОВ <br>
							<span>ВЕБ РАЗРАБОТЧИК</span>
						</h1>
					</div>
				</div>
			</div>
		</div>		
	</main>

	<section class="about" id="about">
		<div class="container">
			<div class="section-item wow animate__animated animate__fadeInUp" >
				<h2 class="section-title">Обо мне</h2>
				<div class="section-title-icon">
					<i class="fas fa-user-tie"></i>
				</div>
			</div>
			<p>Приветствую Вас на своем сайте-портфолио. Я занимаюсь разработкой сайтов Landing Page - одностраничные сайты для Вашего малого бизнеса, для привлечения и увеличения потока клиентов. Сайт сделаю качественно, после моей работы Вам не придется ее переделывать. Одна из причин выбрать меня это то, что я делаю сайты не один, а вместе со своей командой. Срок разработки лендинга: от 5 дней, все зависит от сложности и объема сайта.
			</p>
		</div>
	</section>

	<section class="skills" id="skills">
			<div class="container">
				<div class="section-item wow animate__animated animate__fadeInUp">
					<h2 class="section-title">Мои навыки</h2>
					<div class="section-title-icon">
						<i class="fas fa-cog"></i>
					</div>
				</div>

					<div class="row">
						<div class="col-lg-4 col-md-6 col-sm-12">
							<div class="skills-item">
								<h3 class="skills-subtitle">HTML5/CSS3</h3>
								<ul>
									<li>Валидный, кроссбраузерный код</li>
									<li>Семантичная верстка</li>
									<li>Структутированный код</li>
									<li>Методология БЭМ</li>
								</ul>
							</div>
						</div>
						
						<div class="col-lg-4 col-md-6 col-sm-12">
							<div class="skills-item">
								<h3 class="skills-subtitle">Javascript/JQuery</h3>
								<ul>
									<li>Добавляет интерактивность на Ваш веб-сайт</li>
									<li>Тысячи готовых плагинов</li>
									<li>Компактность кода</li>
										
								</ul>
							</div>
						</div>

						<div class="col-lg-4 col-md-6 col-sm-12">
							<div class="skills-item">
								<h3 class="skills-subtitle">Photoshop/Figma</h3>
								<ul>
									<li>Нарезка изображений из макета</li>
									<li>Экспорт SVG-изображений</li>
									<li>Экспорт стилей для ускоренной верстки</li>
										
								</ul>
							</div>
						</div>

						<div class="col-lg-4 offset-lg-2 col-md-6 col-sm-12">
							<div class="skills-item">
								<h3 class="skills-subtitle">Less</h3>
								<ul>
									<li>Ускоренная стилизация сайта</li>
									<li>Более гибкий код</li>
									<li>Улучшенная логика стилей</li>
									<li>Упрощенный синтаксис</li>
								</ul>
							</div>
						</div>

						<div class="col-lg-5 offset-lg-0 col-md-7 offset-md-3">
							<div class="skills-item">
								<h3 class="skills-subtitle">Bootstrap 4</h3>
								<ul>
									<li>Высокая скорость разработки сайта</li>
									<li>Кроссбраузерность и адаптивность сайта</li>
									<li>Готовые шаблоны элементов сайта</li>
									<li>Оптимизированная навигация</li>
								</ul>
							</div>
						</div>

						
					</div>
				</div>
			</section>	


		<section class="process" id="process">
			<div class="container">
				<div class="section-item  wow animate__animated animate__fadeInUp">
					<h2 class="section-title">Процесс верстки</h2>
					<div class="section-title-icon">
						<i class="fas fa-gavel"></i>
					</div>
				</div>
				<div class="row process-item">
					<div class="col-lg-2 col-md-6 wow animate__animated animate__bounceInRight process-content" data-wow-delay="0.2s">
						<div class="square">
							<div class="square-icon">
								<img src="img/process/scissors.png" alt="" width="100">
							</div>
						</div>
						<div class="square-subtitle">1. Работа с макетом</div>
					</div>

					<div class="col-lg-2 col-md-6 wow animate__animated animate__bounceInRight process-content" data-wow-delay="0.4s">
						<div class="square-subtitle">2. Верстка контента</div>
						<div class="square">
							<div class="square-icon">
								<i class="fab fa-html5"></i>
							</div>
						</div>
					</div>

					<div class="col-lg-2 col-md-6 wow animate__animated animate__bounceInRight process-content" data-wow-delay="0.6s">
						<div class="square">
							<div class="square-icon">
								<i class="fab fa-css3"></i>
							</div>
						</div>
						<div class="square-subtitle">3. Стилизация сайта</div>
					</div>

					<div class="col-lg-2 col-md-6 wow animate__animated animate__bounceInRight process-content" data-wow-delay="0.7s">
						<div class="square-subtitle">4. Адаптивная верстка</div>
						<div class="square">
							<div class="square-icon">
								<i class="fas fa-mobile-alt"></i>
							</div>
						</div>
					</div>

					<div class="col-lg-2 col-md-6 wow animate__animated animate__bounceInRight process-content" data-wow-delay="0.9s">
						<div class="square">
							<div class="square-icon">
								<i class="fab fa-chrome"></i>
							</div>
						</div>
						<div class="square-subtitle">5. Проверка в браузерах</div>
					</div>

					<div class="col-lg-2 col-md-6  wow animate__animated animate__bounceInRight process-content" data-wow-delay="1.1s">
						<div class="square-subtitle">6. Устранение ошибок</div>
						<div class="square">
							<div class="square-icon">
								<i class="fas fa-exclamation-triangle"></i>
							</div>
						</div>
					</div>
				</div>


				<h4 class="process-use">
					Что я использую в процессе верстки
				</h4>
				<div class="process-slider">
						<img src="img/slider/adobe-photoshop.png" >
						<img src="img/slider/less.png" >
						<img class="process-img" src="img/slider/javascript.png">
						<img src="img/slider/html-css.png" >
						<img src="img/slider/unnamed.png" >
						<img src="img/slider/figma.png">
						<img src="img/slider/bootstrap.png" >
				</div>
			</div>

		</section>					


		<section class="portfolio" id="works">
			<div class="container">
				<div class="section-item  wow animate__animated animate__fadeInUp">
					<h2 class="section-title">Портфолио</h2>
					<div class="section-title-icon">
						<i class="fas fa-camera"></i>
					</div>
				</div>
				<div class="portfolio-item">
					<div class="row">
						<div class="col-lg-3 col-md-6 wow animate__animated animate__zoomIn">
							<div class="portfolio-photo">
								<img src="img/portfolio/штукатурка.jpg" alt="" width="310" height="420">
								<div class="portfolio-bg"></div>
								<div class="portfolio-border">
									<p>Препроцессор LESS. Адаптив на Bootstrap. Landing Page</p>
									<a href="https://halilbasir.github.io/Практика/src/" class="btn">Смотреть</a>
									
								</div>
							</div>
						</div>
						<div class="col-lg-3 col-md-6 wow animate__animated animate__zoomIn">
							<div class="portfolio-photo">
								<img src="img/portfolio/mogo.jpg" alt="" width="310" height="420">
								<div class="portfolio-bg"></div>
								<div class="portfolio-border">
									<p>Адаптив с помощью media-запросов. <br>Landing Page</p>
									<a href="https://halilbasir.github.io/MoGo/assets/" class="btn">Смотреть</a>
								</div>
							</div>
						</div>
						<div class="col-lg-3 col-md-6 wow animate__animated animate__zoomIn">
							<div class="portfolio-photo">
								<img src="img/portfolio/ActiveBox.jpg" alt="" width="310" height="420">
								<div class="portfolio-bg"></div>
								<div class="portfolio-border">
									<p>Адаптив с помощью media-запросов. <br>Landing Page</p>
									<a href="https://halilbasir.github.io/ActiveBox/assets/" class="btn">Смотреть</a>
								</div>
							</div>
						</div>
						<div class="col-lg-3 col-md-6 wow animate__animated animate__zoomIn">
							<div class="portfolio-photo">
								<img src="img/portfolio/Axit.jpg" alt="" width="310" height="420">
								<div class="portfolio-bg"></div>
								<div class="portfolio-border">
									<p>Landing Page</p>
									<a href="https://halilbasir.github.io/Axit/" class="btn">Смотреть</a>
								</div>
							</div>
						</div>
						<div class="col-lg-3 col-md-6 wow animate__animated animate__zoomIn">
							<div class="portfolio-photo">
								<img src="img/portfolio/trekking.jpg" alt="" width="310" height="420">
								<div class="portfolio-bg"></div>
								<div class="portfolio-border">
									<p>Адаптив с помощью media-запросов. <br>Landing Page</p>
									<a href="https://halilbasir.github.io/Trekking/" class="btn">Смотреть</a>
								</div>
							</div>
						</div>

						<div class="col-lg-3 col-md-6 wow animate__animated animate__zoomIn">
							<div class="portfolio-photo">
								<img src="img/portfolio/golden.jpg" alt="" width="310" height="420">
								<div class="portfolio-bg"></div>
								<div class="portfolio-border">
									<p>Landing Page</p>
									<a href="https://halilbasir.github.io/Golden/src/" class="btn">Смотреть</a>
								</div>
							</div>
						</div>

						<div class="col-lg-3 col-md-6 wow animate__animated animate__zoomIn">
							<div class="portfolio-photo">
								<img src="img/portfolio/Безымянндддддый.jpg" alt=""width="310" height="420">
								<div class="portfolio-bg"></div>
								<div class="portfolio-border">
									<p>Препроцессор LESS. Адаптив с помощью media-запросов. <br>Сайт-визитка</p>
									<a href="https://halilbasir.github.io/assets/" class="btn">Смотреть</a>
								</div>
							</div>
						</div>

						<div class="col-lg-3 col-md-6 wow animate__animated animate__zoomIn">
							<div class="portfolio-photo">
								<img src="img/portfolio/lapkin.jpg" alt="" width="310" height="420">
								<div class="portfolio-bg"></div>
								<div class="portfolio-border">
									<p>Landing Page</p>
									<a href="https://halilbasir.github.io/lapkin/assets/" class="btn">Смотреть</a>
								</div>
							</div>
						</div>
					</div>

				</div>
			</div>
		</section>


		<section class="contacts" id="contacts">
			<div class="container">
				<div class="section-item  wow animate__animated animate__fadeInUp">
					<h2 class="section-title">Контакты</h2>
					<div class="section-title-icon">
						<i class="fas fa-comment"></i>
					</div>
				</div>

				<div class="contacts-item">
					<div class="row">
						<div class="col-lg-3 col-md-6">
							<div class="contacts-content">
								<div class="contacts-icon contacts-icon1">
									<i class="fas fa-map-marker-alt"></i>
								</div>
								<h4 class="contacts-title">Адрес</h4>
								<a class="contacts-text">Russia</a>
							</div>
						</div>

						<div class="col-lg-3 col-md-6">
							<div class="contacts-content">
								<div class="contacts-icon contacts-icon2">
									<i class="fas fa-mobile-alt"></i>
								</div>
								<h4 class="contacts-title">Phone</h4>
								<a href="tel:+79872533419" class="contacts-text">+79872533419</a>
							</div>
						</div>

						<div class="col-lg-3 col-md-6">
							<div class="contacts-content">
								<div class="contacts-icon contacts-icon3">
									<i class="far fa-envelope"></i>
								</div>
								<h4 class="contacts-title">Email</h4>
								<a href="mailto:rajulshahsov@gmail.com" class="contacts-text">rajulshahsov@gmail.com</a>
							</div>
						</div>
				
						<div class="col-lg-3 col-md-6">
							<div class="contacts-content">
								<div class="contacts-icon contacts-icon4">
									<i class="fab fa-whatsapp"></i>
								</div>
								<h4 class="contacts-title">WhatsApp</h4>
								<a href=" https://api.whatsapp.com/send?phone=79872533419" class="contacts-text">+79872533419</a>
							</div>
						</div>
					</div>
				</div>
			</div>
		</section>

		<footer class="footer">
			<div class="container">
				<div class="footer-icons">
					<a class="footer-icon wow animate__animated animate__rollIn"
					href="https://vk.me/h.basirov">
						<i class="fab fa-vk"></i>
					</a>

					<a class="footer-icon wow animate__animated animate__rollIn"
					 href="https://api.whatsapp.com/send?phone=79872533419" target="_blank">	
						<i class="fab fa-whatsapp"></i>
					</a>

					<a  class="footer-icon wow animate__animated animate__rollIn"
					 href="https://t.me/HalilBasirov" target="_blank">
						<i class="fab fa-telegram-plane"></i>
					</a>	
				</div>

				<p class="footer-develop wow animate__animated animate__fadeInUp">
					<i class="far fa-copyright"></i>
					<span>Portfolio</span> 2020 - Developed by <span>Khalil Basirov</span>
				</p>
			</div>
		</footer>

		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
		<script src="js/app.js"></script>
		<!-- Стили slick slider -->
		<link rel="stylesheet" href="slick/slick.css">
		<link rel="stylesheet" href="slick/slick-theme.css">

		<!-- Библиотека slick slider -->
		<script src="slick/slick.min.js"></script>

		<!-- Инициализация slider -->
		<script>
			$(document).ready(function(){
			  $('.process-slider').slick({
			 	loop: true,
			 	slidesToShow: 4,
			 	slidesToScroll: 1,
			 	responsive: [
			 		{
			 	      breakpoint: 1950,
			 	      settings: {
			 	        slidesToShow: 4,
			 	        prevArrow: '<button class="prev arrow"></button>',
			 	        nextArrow: '<button class="next arrow"></button>',
			 	        slidesToScroll: 1
			 	      }
			 	    },
			 	    {
			 	      breakpoint: 991,
			 	      settings: {
			 	        slidesToShow: 3,
			 	        prevArrow: '<button class="prev arrow"></button>',
			 	        nextArrow: '<button class="next arrow"></button>',
			 	        slidesToScroll: 1
			 	      }
			 	    },
			 	    {
			 	      breakpoint: 767,
			 	      settings: {
			 	        slidesToShow: 2,
			 	        prevArrow: '<button class="prev arrow"></button>',
			 	        nextArrow: '<button class="next arrow"></button>',
			 	        slidesToScroll: 1
			 	      }
			 	    },
			 	     {
			 	      breakpoint: 567,
			 	      settings: {
			 	        slidesToShow: 1,
			 	        prevArrow: '<button class="prev arrow"></button>',
			 	        nextArrow: '<button class="next arrow"></button>',
			 	        slidesToScroll: 1
			 	      }
			 	    }
			 	  ]
			  });
			});
		</script>
		<script src="js/wow.min.js"></script>

		<script>
		new WOW().init();
		</script>
</body>
</html>

