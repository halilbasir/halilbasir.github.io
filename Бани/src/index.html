<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>77</title>
	<link rel="stylesheet" href="css/bootstrap.css">
	<link rel="stylesheet" href="css/style.css">

	<link href="https://fonts.googleapis.com/css2?family=Cardo:ital@1&family=Open+Sans:wght@700&family=Raleway:wght@400;600;700&family=Montserrat:wght@400;500;600;700&family=IBM+Plex+Sans:wght@500;600;700display=swap" rel="stylesheet">

	<link rel="preconnect" href="https://fonts.gstatic.com">
	<link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:wght@500;600;700&display=swap" rel="stylesheet">

	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="
	sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
</head>
<body>
	<header class="header" id="header">
		<div class="container">
			<div class="header-inner">
				<div class="header-logo">
					<img class="header-logo-img" src="img/logo.png" alt="">
				</div>
				<div class="row">
					
						<nav class="nav navbar navbar-expand-lg" id="nav">
						  <a href="#" class="navbar-brand" data-scroll="#about" >О компании</a>
						  <a href="#" class="navbar-brand" data-scroll="#features" >Преимущества наших бань</a>
						  <a href="#" class="navbar-brand" data-scroll="#process" >Процесс работы</a>
						  <a href="#" class="navbar-brand" data-scroll="#contact" >Контакты</a>
						</nav>
					

					<button class="nav-toggle" id="nav_toggle" type="button">
						<span class="nav-toggle__item">Menu</span>
					</button>
					
					<div class="col-xl-4 offset-xl-10 col-lg-6 offset-lg-4 col-md-12 offset-md-6 col-sm-12 offset-sm-3">
						<div class="header-phone">
							<img src="img/about/Слой 1.png" width="30" alt="">
							<a href="tel:+7 919 633-55-99">+7 919 633-55-99</a>
						</div>
					</div>
				</div>
			</div>
		</div>
	</header>
	<section class="intro" id="intro">
		<div class="container">
			<div class="intro-item">
				<h1 class="intro-title">
					Строительство бань под ключ<br>
				<span>в Набережных Челнах и Татарстане</span>
				</h1>
				
				<div class="intro-ul">
					<ul>
						<li>Мы на рынке с 2013 года.</li>
						<li>Поэтапная оплата, фиксированные цены, скидки и акции</li>
						<li>Профессиональные строители</li>
						<li>Срок возведения бани от 20 дней</li>
						<li>Работаем по договору</li>
					</ul>
				</div>
			</div>
		</div>
	</section>

	<section class="slider">
		<div class="container">
			<h3 class="slider-header">Бани 3х5 под ключ</h3>

			<div class="fotorama" data-keyboard="true" data-swipe="false" data-height="600" data-loop="true" data-autoplay="3000" data-allowfullscreen="true">
					<img src="img/slider/3.jpeg" width="600">
					<img src="img/slider/2.jpeg" width="600">
					<img src="img/slider/1.jpeg" width="600">
					<img src="img/slider/4.jpeg" width="600">
					<img src="img/slider/5.jpeg" width="600">
					<img src="img/slider/6.jpeg" width="600">
					<img src="img/slider/7.jfif" width="600">
					<img src="img/slider/7.jpg" width="600" alt="">
					<img src="img/slider/12.jpg" width="600">
					<img src="img/slider/9.jpg" alt="">
					<img src="img/slider/10.jpg" width="600" alt="">
					<img src="img/slider/11.jpg" width="600" alt="">
			</div>
		</div>
	</section>

	<section class="features" id="features">
		<div class="features-item">
			<div class="features-title">
				<h2>Преимущества наших бань</h2>
				<h3>Закажите строительство в нашей компании и вы получите</h3>
			</div>
				
				<div class="row">
					<div class="col-xl-3 offset-xl-1 col-lg-3 offset-lg-1 col-md-4 offset-md-0 col-sm-6">
						<div class="features-text-1">
							<h4>Доступные цены</h4>
							<p>Хорошие скидки на стройматериалы позволяют строить бани дешевле чем другие компании</p>
						</div>
					</div>
					
					<div class="col-xl-3 offset-xl-4 col-lg-3 offset-lg-4 col-md-4 offset-md-4 col-sm-6">
						<div class="features-text">
							<h4>Всесезонность строительства</h4>
							<p>Для строительства бань нет понятия «строительный сезон» - строить можно даже зимой</p>
						</div>
					</div>
					
					<div class="features-img">
						<img src="img/features.jpg" width="450" alt="">
					</div>	
		
					<div class="col-xl-3 offset-xl-1 col-lg-3 offset-lg-1 col-md-4 offset-md-0 col-sm-6">
						<div class="features-text-1">
							<h4>Долговечность</h4>
							<p>Используем правильные технологии строительства, при необходимости пропитываем материалы огнебиозащитой, что позволяет служить бане 50 лет и более</p>
						</div>
					</div>

					<div class="col-xl-3 offset-xl-4 col-lg-3 offset-lg-4 col-md-4 offset-md-4 col-sm-6">
						<div class="features-text">
							<h4>Быстровозводимость</h4>
							<p>Средний срок строительства - 20 дней, включая фундамент</p>
						</div>
					</div>				
					
									
					<div class="col-xl-3 offset-xl-1 col-lg-3 offset-lg-1 col-md-4 offset-md-0 col-sm-6">
						<div class="features-text-1">
							<h4>Скидки и акции</h4>
							<p>Мы постоянно проводим различные акции, даем приятные бонусы при заказе строительства в нашей компании</p>
						</div>
					</div>

									
					<div class="col-xl-3 offset-xl-4 col-lg-3 offset-lg-4 col-md-4 offset-md-4 col-sm-6">
						<div class="features-text">
							<h4>Грамотный и продуманный проект</h4>
							<p>Построим баню по нашим готовым проектам, либо разработаем индивидуальный по вашим пожеланиям</p>
						</div>
					</div>				
				</div>	
		</div>
	</section>

	<section class="about" id="about">
		<div class="container">
			<h3 class="about-title">О компании</h3>
			<div class="row">
				<div class="col-xl-4 offset-xl-0 col-lg-4 offset-lg-3 col-md-4 offset-md-2">
					<img class="about-img" src="img/about.jfif" alt="" width="700">
				</div>
				<div class="col-xl-6 col-lg-8 offset-lg-2 col-md-12">
					<div class="about-text">
						<p>Срочно нужна баня под ключ в Татарстане? Звоните нам мы построим вам хорошую семейную баню недорого. Наш средний срок строительства 20 дней. Мы строим бани из разных материалов — бревен различного диаметра и бруса с различным сечением. По желанию клиента можем внести индивидуальные коррективы. В нашей команде работают профессионалы, любящие свое дело, поэтому к каждому объекту мы подходим с душой. Выполняем строительство как и полностью готовых бань под ключ, так и бань без отделки.</p>
					</div>
				</div>
				
			</div>
		</div>
	</section>

	<section class="process" id="process">
		<div class="container">
			<div class="row">
				<div class="about-work-1 col-xl-2 col-lg-3 col-md-3 col-sm-4 col-12">
					<div class="about-item">
						<div class="about-work">
							<img src="img/about/1.png" alt="">
							<span class="about-numeral">1</span>
							<p>Звонок</p>
						</div>
					</div>
				</div>

				<div class="col-xl-2 col-lg-3 col-md-3 col-sm-4 col-12">
					<div class="about-item">
						<div class="about-work">
							<img src="img/about/2.png" alt="">
							<span class="about-numeral">2</span>
							<p>Выезд на участок</p>
						</div>
					</div>
				</div>

				<div class="col-xl-2 col-lg-3 col-md-3 col-sm-4 col-12">
					<div class="about-item">
						<div class="about-work">
							<img src="img/about/3.png" alt="">
							<span class="about-numeral">3</span>
							<p>Заливка фундамента</p>
						</div>
					</div>
				</div>

				<div class="col-xl-3 col-lg-3 col-md-3 col-sm-4 col-12">
					<div class="about-item">
						<div class="about-work">
							<img src="img/about/4.png" alt="">
							<span class="about-numeral">4</span>
							<p>Строительство коробки</p>
						</div>
					</div>
				</div>

				<div class="col-xl-2 offset-xl-3 col-lg-3 col-md-3 col-sm-4 col-12">
					<div class="about-item">
						<div class="about-work">
							<img src="img/about/5.png" alt="">
							<span class="about-numeral">5</span>
							<p>Кровельные работы</p>
						</div>
					</div>
				</div>

				<div class="col-xl-2 col-lg-3  col-md-3 col-sm-4 col-12">
					<div class="about-item">
						<div class="about-work">
							<img src="img/about/6.png" alt="">
							<span class="about-numeral">6</span>
							<p>Отделка</p>
						</div>
					</div>
				</div>

				<div class="col-xl-3 col-lg-3 col-md-3 col-sm-4 col-12">
					<div class="about-item">
						<div class="about-work">
							<img src="img/about/7.png" alt="">
							<span class="about-numeral">7</span>
							<p>Сдача объекта</p>
						</div>
					</div>
				</div>
			</div>
		</div>
	</section>
	
	<footer class="footer" id="contact">
		<div class="container">
			<div class="footer-item">
				<div class="col-lg-6 col-md-3 col-sm-4">
					<div class="footer-icons">
						<a class="footer-icon"
						href="https://vk.me/h.basirov">
							<i class="fab fa-vk"></i>
						</a>

						<a class="footer-icon"
						 href="https://api.whatsapp.com/send?phone=+7 919 633-55-99" target="_blank">	
							<i class="fab fa-whatsapp"></i>
						</a>

						<a class="footer-icon"
						 href="https://t.me/MaratabuSalman" target="_blank">
							<i class="fab fa-telegram-plane"></i>
						</a>
						
					</div>
				</div>
				<div class="col-lg-6 col-md-3 offset-md-1 col-5">
					<div class="footer-develop">
					<span>Разработал сайт</span>
					<a href="file:///G:/My_Projects/Projects/Portfolio/src/index.html">Khalil Basirov</a>
					</div>	
				</div>
			</div>
		</div>
	</footer>

	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
	<script src="js/app.js"></script>

	<link href="https://cdnjs.cloudflare.com/ajax/libs/fotorama/4.6.4/fotorama.css" rel="stylesheet">
	<script src="https://cdnjs.cloudflare.com/ajax/libs/fotorama/4.6.4/fotorama.js"></script>
</body>
</html>