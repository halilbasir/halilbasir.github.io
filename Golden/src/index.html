<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="style.css">
<body>
	<header>
		<div class="container">
			<div class="heading сlearfix">
				<img src="img/logo.png" alt="golden" class="logo">
				<nav>
					<ul class="menu">
						<li>
							<a href="#">Home</a>
						</li>
						<li>
							<a href="#">Services</a>
						</li>
						<li>
							<a href="#">Portfolio</a>
						</li>
						<li>
							<a href="#">About</a>
						</li>
						<li>
							<a href="#">Contact</a>
						</li>
					</ul>
				</nav>
			</div>
			<div class="titles_first">
				Welcome To Our Studio!
			</div>
			<h1>
				It’s nice to Meet you
			</h1>
			<div >
				<a class="button" href="#"> 
					Tell Me More 
				</a>
			</div>	
	    </div>
	</header>

	<section id="services">
		<div class="container">
			<div class="titles">
				<h2>Services</h2>
				<p>Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="services clearfix">
				<div class="services_item">
					<img src="img/icon1.png" alt="icon1">
					<h3>E-Commerce</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			    <div class="services_item">
					<img src="img/icon2.png" alt="icon2">
					<h3>Responsive Web</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
                </div>
				<div class="services_item">
					<img src="img/icon3.png" alt="icon3">
					<h3>Web Security</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			</div>
		</div>
	</section>

	<section id="portfolio">
		<div class="container">
			<div class="titles">
				<div class="yt">
					<h2>our portfolio</h2>
					<p>Proin iaculis purus consequat sem cure.</p>
				</div>
			</div> 
			<div class="works clearfix">	
				<img src="img/img1.jpg" alt="">
				<img src="img/img2.jpg" alt="">
				<img src="img/img3.jpg" alt="">
			</div>
		</div>	
	</section>
	<section id="about">
		<div class="container">
			<div class="titles">
			   <h2>about us</h2>
			   <p>Proin iaculis purus consequat sem cure.</p>
			</div>

 		<div class="texts">
 			<div class="read"></div>
				<div class="about1">
					<h3>July 2010</h3>
					<h4>Our Humble Beginnings</h4>
					<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<img src="img/about1.jpg" alt="" class="center-img1">
   	 	</div>

    	<div class="read1">
			<div class="about2 ">
				<h3>January 2011</h3>
				<h4>Facing Startup Battles</h4>
				<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
			</div>
				<img src="img/about2.jpg" alt="" class="center-img"><h3></h3>
		</div>
		<div class="read1">
			<div class="about3">
				<h3>December 2012</h3>
				<h4>Enter The Dark Daysfebruary 2014</h4>
				<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			<img src="img/about3.jpg" alt="" class="center-img1"><h3></h3>
	</div>
	
	<div class="read1">
		<div class="about4">
				<h3>FEBRUARY 2014</h3>
				<h4>Our Triumph</h4>
				<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
		</div>
    	<img src="img/about4.jpg" alt="" class="center-img"><h3></h3>
	</div>
    </div>
</section>

	<footer>
		<div class="container">
			<div class="end"><p>©Copyright 2014 FreebiesXpress.com</p></div>
	    </div>	 
	</footer>
</body>
</html>